db-scripts/src/main/resources/migration.sql

```diff
@@ -567,3 +567,17 @@ UPDATE `info` SET `DB_SCHEMA_VERSION`="2.7.4";
 ##version: 2.8.0
 ALTER TABLE `cancer_study` MODIFY COLUMN `PMID` varchar(1024) DEFAULT NULL;
 UPDATE `info` SET `DB_SCHEMA_VERSION`="2.8.0";
+
+##version: 2.8.1
+ALTER TABLE `mutation_event` DROP INDEX `CHR`;
```

Comment:
We did not explicitly name this key in cgds.sql, but mysql seems to have given it the same name as the first field (CHR). I could not find this "default key name" behavior defined in the mysql reference documentation. This is similar to the problem with our (unnamed) foreign key constraints -- it is not certain that we will be able to modify them properly later. But I think your migration code is the best that we can do without getting really "hacky" ... such as dumping the table schema to a text string and then do string parsing to locate the key and discover the index name.

Reply:
Will the import be much slower now? There will be many queries to `mutation_event` table, and now there is no index anymore. Can you please show a comparison of loading time for a regular study (say BRCA_TCGA) with *vs* without this index on a system that already has a reasonably large set of studies (perhaps on a copy of the public portal's DB)? Perhaps we need to add some indexes to compensate the performance loss (if any)?

Reply:
@pieterlukasse I would think the opposite - without having to write to an index, the import would be faster, no?